mozartdata/transforms/goodr_reporting/goodr_com_usage_kpis.sql

```diff
@@ -0,0 +1,198 @@
+                                   , shopify."SESSIONS THAT COMPLETED CHECKOUT" as sess_completed_checkout
+                                   , shopify."CONVERSION RATE"                  as daily_conversion_rate
+                                 from
+                                     dim.date                                       as d
+                                     inner join
+                                         shopify_exports.kpi_data_20240101_20250206 as shopify
+                                             on
+                                             d.date = shopify.day::date
+                                 where
+                                       d.date >= date_trunc('year', current_date)
+                                   and d.date <= current_date
+                             )
+  , shopify_data_last_ytd as (
+                                 select
+                                     d.date::date                               as date
+                                   , shopify.sessions
+                                   , shopify."ONLINE STORE VISITORS"            as users
+                                   , shopify."NEW CUSTOMERS"                    as new_customers
+                                   , shopify.customers                          as total_customers
+                                   , shopify."SESSIONS THAT COMPLETED CHECKOUT" as sess_completed_checkout
+                                   , shopify."CONVERSION RATE"                  as daily_conversion_rate
+                                 from
+                                     dim.date                                       as d
+                                     inner join
+                                         shopify_exports.kpi_data_20240101_20250206 as shopify
+                                             on
+                                             d.date = shopify.day::date
+                                 where
+                                       d.date >= date_trunc('year', dateadd(year, -1, current_date))
+                                   and d.date <= dateadd(year, -1, current_date)
+                             )
+  , media_totals_this_ytd as (
+                                 select
+                                     date
+                                   , sum(spend)       as total_spend
+                                   , sum(revenue)     as total_revenue
+                                   , sum(impressions) as total_impressions
+                                   , sum(clicks)      as total_clicks
+                                 from
+                                     goodr_reporting.performance_media as pm
+                                 where
+                                       lower(pm.account_country) = 'usa'
+                                   and date >= date_trunc('year', current_date)
+                                   and date <= current_date
```

Comment:
same idea as above. just being consistent. you can ignore this one.
```suggestion
                                   and  date_trunc('year', date) = date_trunc('year', current_date)
```

Reply:
Ah but I'm weird and worried about dates that might be in the future. The method I have prevents that.